packages/radix-vue/src/Combobox/ComboboxRoot.vue

```diff
@@ -211,10 +211,10 @@ watch(stringifiedModelValue, async () => {
   immediate: !props.searchTerm,
 })
 
-watch(() => filteredOptions.value.length, async (length) => {
+watch(() => [filteredOptions.value.length, searchTerm.value.length], async ([length, searchTermLength], [oldLength, oldSearchTermLength]) => {
```

Comment:
If searchTerm.value is null/undefined this causes a .length error and bugs out the Combobox